earth_enterprise/rpms/build.gradle

```diff
@@ -355,10 +357,12 @@ task openGeeServerRpm (type: GeeRpm, dependsOn: openGeeSharedFiles) {
     // Prefix variable definitions to all install scripts:
     installUtils = file('build/shared/install-utils.sh')
 
+    // Add geresetpgdb script:
+    installUtils = file(new File(stagedInstallDir_server_opt, 'google/bin/gepgcommon'))
+
     preInstall = file('opengee-server/pre-install.sh')
     postInstall = file('opengee-server/post-install.sh')
     preUninstall = file('opengee-server/pre-uninstall.sh')
-    postUninstall = file('opengee-server/post-uninstall.sh')
```

Comment:
Q: Aren't there still post-unistall scripts? I don't see them removed in this pull request.